Styles/ECOWAS_Generator_Points_GCS_WGS84_WAPP_20230609.sld

```diff
@@ -0,0 +1,40 @@
+  <NamedLayer>
+    <se:Name>ECOWAS_Generator_Points_GCS_WGS84_WAPP</se:Name>
+    <UserStyle>
+      <se:Name>ECOWAS_Generator_Points_GCS_WGS84_WAPP</se:Name>
+      <se:FeatureTypeStyle>
+        <se:Rule>
+          <se:Name>Single symbol</se:Name>
+          <se:PointSymbolizer>
+            <se:Graphic>
+              <!--Parametric SVG-->
+              <se:ExternalGraphic>
+                <se:OnlineResource xlink:type="simple" xlink:href="base64:PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48IS0tIFVwbG9hZGVkIHRvOiBTVkcgUmVwbywgd3d3LnN2Z3JlcG8uY29tLCBHZW5lcmF0b3I6IFNWRyBSZXBvIE1peGVyIFRvb2xzIC0tPg0KPHN2ZyBmaWxsPSIjMDAwMDAwIiB3aWR0aD0iODAwcHgiIGhlaWdodD0iODAwcHgiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJNMTIsMkExMCwxMCwwLDEsMCwyMiwxMiwxMCwxMCwwLDAsMCwxMiwyWm00LjM5LDE0LjlhMTEuNjcsMTEuNjcsMCwwLDEtMS45NC40NywxMi45MiwxMi45MiwwLDAsMS0xLjg2LjEzLDQuNzcsNC43NywwLDAsMS0zLjcxLTEuNDJBNS44OSw1Ljg5LDAsMCwxLDcuNjEsMTIsNS4zOSw1LjM5LDAsMCwxLDkuMDksOCw1LjYxLDUuNjEsMCwwLDEsMTMuMiw2LjVhNy45MSw3LjkxLDAsMCwxLDMuMTguNjZMMTUuNjMsOWE1LjQ0LDUuNDQsMCwwLDAtMi40NC0uNTgsMywzLDAsMCwwLTIuMzYsMUEzLjg1LDMuODUsMCwwLDAsOS45NCwxMmE0LjI4LDQuMjgsMCwwLDAsLjcyLDIuNjcsMi40NiwyLjQ2LDAsMCwwLDIuMDcuOTEsNy4wNiw3LjA2LDAsMCwwLDEuNDUtLjE0VjEzLjIzaC0ybDAtMS44OWg0LjI0WiIvPjwvc3ZnPg==?fill=%23ff0000&amp;fill-opacity=1&amp;outline=%23232323&amp;outline-opacity=1&amp;outline-width=0"/>
+                <se:Format>image/svg+xml</se:Format>
+              </se:ExternalGraphic>
+              <!--Plain SVG fallback, no parameters-->
+              <se:ExternalGraphic>
+                <se:OnlineResource xlink:type="simple" xlink:href="base64:PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48IS0tIFVwbG9hZGVkIHRvOiBTVkcgUmVwbywgd3d3LnN2Z3JlcG8uY29tLCBHZW5lcmF0b3I6IFNWRyBSZXBvIE1peGVyIFRvb2xzIC0tPg0KPHN2ZyBmaWxsPSIjMDAwMDAwIiB3aWR0aD0iODAwcHgiIGhlaWdodD0iODAwcHgiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJNMTIsMkExMCwxMCwwLDEsMCwyMiwxMiwxMCwxMCwwLDAsMCwxMiwyWm00LjM5LDE0LjlhMTEuNjcsMTEuNjcsMCwwLDEtMS45NC40NywxMi45MiwxMi45MiwwLDAsMS0xLjg2LjEzLDQuNzcsNC43NywwLDAsMS0zLjcxLTEuNDJBNS44OSw1Ljg5LDAsMCwxLDcuNjEsMTIsNS4zOSw1LjM5LDAsMCwxLDkuMDksOCw1LjYxLDUuNjEsMCwwLDEsMTMuMiw2LjVhNy45MSw3LjkxLDAsMCwxLDMuMTguNjZMMTUuNjMsOWE1LjQ0LDUuNDQsMCwwLDAtMi40NC0uNTgsMywzLDAsMCwwLTIuMzYsMUEzLjg1LDMuODUsMCwwLDAsOS45NCwxMmE0LjI4LDQuMjgsMCwwLDAsLjcyLDIuNjcsMi40NiwyLjQ2LDAsMCwwLDIuMDcuOTEsNy4wNiw3LjA2LDAsMCwwLDEuNDUtLjE0VjEzLjIzaC0ybDAtMS44OWg0LjI0WiIvPjwvc3ZnPg=="/>
+                <se:Format>image/svg+xml</se:Format>
+              </se:ExternalGraphic>
+              <!--Well known marker fallback-->
```

Comment:
If we get the SVG working properly there is no need for a fall back part of the SVG